### PR TITLE
fix: update Resources nav according to dev user journeys doc

### PR DESCRIPTION
**Pull request by ffe9f8**
Update according to the [Developer Documentation User Journeys](https://docs.google.com/document/d/1f1fr4wo6_GD6JbwNqhLTMbWxB2FLwCmHerxea7LwKwU/edit#heading=h.qjcmu7yubf0s) doc.

### [New section] Addresses
- Mainnet
- Testnet

### [New section] RPCs
- RPC Endpoints
- Free Public RPC (change from “Using ipRPC (Free Public RPC)”)

### [New section] Tokens
- Stake AXL tokens (move out of “setup" section in Validators, currently https://docs.axelar.dev/validator/setup/stake-axl) 
- Wrap / unwrap tokens

### [New section] Axelarscan
- Add account to Axelarscan

### [New section]  Helpers (change name)
- Satellite
- MetaMask
- Add network to Keplr wallet

### [New section] Community
- Community pool proposals
- Bug bounty

### CODE REVIEW

File: src/layouts/navigation.ts [hunk 1]
```diff
@@ -307,20 +306,48 @@ export const getNavigation = (section) => {
   nav.push({
     header: "Resources",
     children: [
-      { title: "Mainnet", href: "/resources/mainnet" },
-      { title: "Testnet", href: "/resources/testnet" },
-      { title: "RPC Endpoints", href: "/resources/" },
       {
-        title: "Community Pool Proposals",
-        href: "/resources/governance/community-pool-proposals",
+        title: "Addresses",
```
Comment (Olanetsoft) [inline src/layouts/navigation.ts:310]: Is this okay as "Addresses" ? Maybe we can change it to "Contract Addresses"

**Review by Olanetsoft — COMMENTED** [review]
Aside this feedback. LGTM.

File: src/layouts/navigation.ts (hunk 1 above)
Comment (ffe9f8) [reply src/layouts/navigation.ts:310]: Sounds good to me.

**Comment by ffe9f8** [comment]
**Note:** Each section is its own folder (i.e. Every file in the "Contract addresses" section should be in the folder `/contract-addresses`) except for the "misc" section, since the purpose of that section is to consolidate all the miscellaneous docs living in the `/resources` folder.

**Review by Olanetsoft — APPROVED** [review]

**Pull request merged**